cmd/transcriber/call/tracks.go

```diff
@@ -115,6 +115,31 @@ func (t *Transcriber) processLiveTrack(track trackRemote, sessionID string, user
 	}
 	defer oggWriter.Close()
 
+	// Live captioning:
```

Comment:
I think there's a chance to decouple further if we run this logic in its own goroutine. Two things I'd like to see:

- Avoid decoding as part of the reading loop since it's a blocking operation. We should pass the data to a channel and deal with the rest asynchronously.
- Have a config option/flag to entirely disable live captions and only run the basic post-call transcriber. This is important since it will allow our system admins some flexibility especially since there are potentially significant performance considerations to be made.

Reply:
Good suggestions, thank you.
- Done, and then I also decoupled the reading packets/decoding into PCM stage from the VAD/transcription phase. This removes the dependency between those steps (the `window` slice), and means we don't have to worry about a pathological case where the server just can't transcribe fast enough. Before, that could have led to blocking the packet reading goroutine, which would fill up the payload buffer, which would have led to blocking the reading loop. Now it won't -- all that will happen is we will add more to the `toBeTranslated` slice, which is fine. We may drop untranslated voice, but that's what we should do anyway (if the server is being crushed).
- I've added config options (a couple to the system console, and one in the env vars -- see the plugin changes).